performance_tests/clickhouse_vs_greptimedb_comparison.js

```
@@ -0,0 +1,490 @@
+      console.log(`  ClickHouse Observations: ${clickhouseResults.insert[size]?.observations || 'N/A'}`);
+      console.log(`  ClickHouse Scores:      ${clickhouseResults.insert[size]?.scores || 'N/A'}`);
+      console.log(`  GreptimeDB Traces:      ${greptimeResults.insert[size]?.traces || 'N/A'}`);
+    }
+    
+    console.log('\n📈 QUERY PERFORMANCE (milliseconds):');
+    console.log('------------------------------------');
+    const commonQueries = Object.keys(clickhouseResults.query).filter(q => 
+      greptimeResults.query.hasOwnProperty(q)
+    );
+    
+    for (const queryName of commonQueries) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)